pontoon/base/models.py

```diff
@@ -152,6 +153,15 @@ class Locale(models.Model):
 
     team_description = models.TextField(blank=True)
 
+    #: Most recent translation approved or created for this locale.
+    latest_translation = models.ForeignKey(
+        'Translation',
+        blank=True,
+        null=True,
+        related_name='+',
+        on_delete=models.SET_NULL
```

Comment:
That thing up there means I've never seen on_delete before. But I'm also wondering, what would actually change if latest_translation got deleted in a cascade with the foreign key? Seems like you have that case covered in get_latest_activity().